### PR TITLE
Add NSJ Screw into tunnels logic to Abandoned Worksite [Echoes]

### PR DESCRIPTION
**Pull request by f-raZ0R**
This pull request adds logic to the echoes database, to access the pickup in Abandoned Worksite, from the door in Great Bridge, with solely the Screw Attack. This uses the "Screw Attack into Tunnels/Openings" trick, at difficulty level advanced. A video of the trick being performed multiple times in succession, can be seen here:
https://github.com/randovania/randovania/assets/30442287/cfbf3bad-4f6e-4804-8ced-84a6a3e10153

### CODE REVIEW

**Review by henriquegemignani — CHANGES_REQUESTED** [review]
Could you upload the video to youtube and add as comment to the connection?

**Comment by f-raZ0R** [comment]
Done

File: randovania/games/prime2/json_data/Torvus Bog.txt [hunk 1]
```diff
@@ -698,6 +698,9 @@ Extra - in_dark_aether: False
                   # https://www.youtube.com/watch?v=pJlUeNML-vo No Space Jump
                   Bomb Space Jump (Advanced)
                   Space Jump Boots and Bomb Space Jump (Intermediate)
+          All of the following:
+              # https://youtu.be/RSIbffrfJaE
```
Comment (dyceron) [inline randovania/games/prime2/json_data/Torvus Bog.txt:702]: Video cuts off before reaching the ledge. An updated video that shows this would be nice.

**Review by dyceron — CHANGES_REQUESTED** [review]

File: randovania/games/prime2/json_data/Torvus Bog.txt (hunk 1 above)
Comment (f-raZ0R) [reply randovania/games/prime2/json_data/Torvus Bog.txt]: I uploaded a new video, didn't realize that it got cut off, whoops!

**Review by dyceron — APPROVED** [review]

**Pull request merged**